### PR TITLE
SONARPY-1342 Improve typeshed unit test to avoid serializing Typeshed

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
All this PR does is using some sample files of `Typeshed` for the unit tests, instead of the full thing.
On my machine, this reduces the time taken by the unit tests from ~50s to ~10s. And I don't think we lose any meaningful confidence in the implementation by using the subset of `Typeshed` rather than the complete library.

### CODE REVIEW

File: python-frontend/typeshed_serializer/tests/conftest.py [hunk 1]
```diff
@@ -31,6 +31,8 @@
 
 @pytest.fixture(scope="session")
 def typeshed_stdlib():
+    typeshed_serializer.STDLIB_PATH = "resources/mock_stdlib"
```
Comment (joke1196) [inline python-frontend/typeshed_serializer/tests/conftest.py:34]: This is the way, with the fixtures :+1:

File: python-frontend/typeshed_serializer/tests/conftest.py [hunk 2]
```diff
@@ -31,6 +31,8 @@
...
+    typeshed_serializer.CURRENT_PATH = os.path.dirname(__file__)
```
Comment (joke1196) [inline python-frontend/typeshed_serializer/tests/conftest.py:35]: Here can we use the CURRENT PATH constant instead? Genuine question not sure when the fixture is invoked.

**Review by joke1196 — APPROVED** [review]
LGTM! Just added a small question. Thank you for this; the speed upgrade feels really nice!

File: python-frontend/typeshed_serializer/tests/conftest.py (hunk 2 above)
Comment (guillaume-dequenne-sonarsource) [reply python-frontend/typeshed_serializer/tests/conftest.py]: Thanks, good catch! I somehow missed it.

**Pull request merged**